sdk/formrecognizer/azure-ai-formrecognizer/tests/test_compose_model_async.py

```diff
@@ -79,3 +95,12 @@ async def test_compose_continuation_token(self, client, container_sas_url):
             self.assertIsNotNone(result)
 
             await initial_poller.wait()  # necessary so azure-devtools doesn't throw assertion error
+
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalClientPreparer(training=True, client_kwargs={"api_version": "2.0"})
+    async def test_compose_model_bad_api_version(self, client, container_sas_url):
+        async with client:
+            with pytest.raises(ValueError) as excinfo:
+                poller = await client.begin_create_composed_model(["00000000-0000-0000-0000-000000000000", "00000000-0000-0000-0000-000000000000"])
+                result = await poller.result()
+            assert "API version 2.0 does not have operation 'begin_compose_custom_models_async'" in str(excinfo.value)
```

Comment:
I think you should wrap this error (i.e. catch it and throw a different error), since we've wrapped the method name `begin_compose_custom_models_async`